shared/types/index.ts

```diff
@@ -15,6 +15,8 @@ export interface CustomFormData {
   miscellaneous: {
     // This is a placeholder for any additional data that the user wants to include in the invoice
     labels: string[];
+    builderId: string;
+    createdWith: string;
```

Comment:
Please try extending the rnf_invoice 0.0.3 types instead of redefining them with the additional miscellaneous properties.

Reply:
Done :white_check_mark:

Reply:
Fixed, the types are extended